content/SCALE/SCALECLIReference/Account/CLIuser.md

```diff
@@ -199,84 +196,60 @@ account user get_next_uid
-
-The `sid_info` option retrieves the Security Identifier (SID) and domain information for the user.
-In some instances retrieving SID and domain information makes the operation hang until the winbindd request timeout has been reached if the winbindd connection manager has not yet marked the domain as offline. Check the Active Directory service state prior to batch operations using this option.
+The `get_user_obj` command has one required property, `get_user_obj` that expects you to enter property arguments formatted as an array.
+Enter the command string, using `=` followed by the curly brackets that enclose the property argument. 
+You can enter either the `uid` or `username` property in this command string.
+Enter the property enclosed in double quotes, then the `:` delimiter followed by the value enclosed in double quotes, with no space after the colon. Press<kbd>Enter</kbd>.
+The command returns a table for the UID that includes the **pw_name**, **pw_uid**, **pw_gid**, **pw_gecos**, **pw_dir**, and **pw_shell** values.
 
 #### Usage
 
 From the CLI prompt, enter:
 
 <code>account user get_user_obj get_user_obj={"uid": <i>3001</i>}</code>
 
-Press <kbd>Enter</kbd>
-
-From the **account** prompt, enter:
-
-<code>user get_user_obj get_user_obj={"uid": <i>3001</i>}</code>
+Where *3001* is the id number for the targeted account.
 
-Press <kbd>Enter</kbd>
+Alternately enter:
 
-Where *3001* is the id number for the targeted account.
+<code>get_user_obj={"username":"<i>testuser</i>"}</code>
 
-You can also use <code>get_user_obj={"username": "<i>testuser</i>"}</code>, where *testuser* is the username of the targeted account.
+Where *testuser* is the username of the targeted account.
 
 {{< expand "Command Example" "v" >}}
-<pre><code>
-account user get_user_obj get_user_obj={"username": "<i>testuser</i>", "get_groups": <i>true</i>, "sid_info": <i>true</i>}
+<code>
+account user get_user_obj get_user_obj={"username": "<i>testuser</i>"}
 +-----------+--------------+
 |   pw_name | testuser     |
 |    pw_uid | 3000         |
 |    pw_gid | 65534        |
 |  pw_gecos | Test User    |
 |    pw_dir | /nonexistent |
 |  pw_shell | /usr/bin/zsh |
-| grouplist | 65534        |
-|           | 3000         |
-|           | 3009         |
-|           | 545          |
-|           | 3022         |
-|           | 3002         |
-|  sid_info | &lt;null&gt;       |
 +-----------+--------------+
 </code></pre>
 {{< /expand >}}
 {{< /expand >}}
 
-
 ### Has_Local_Administrator_Set_Up Command
 
 The `has_local_administrator_set_up` command returns whether a local administrator account with a valid password exists.
 
 {{< expand "Using the Has_Local_Administrator_Set_Up Command" "v" >}}
-
 #### Description
-
 The `has_local_administrator_set_up` command does not require entering properties or values.
+Enter the command, then press<kbd>Enter</kbd>.
+The command returns true if the admin account has a password, false if it does not.
```

Comment:
Technically I think it returns true if either admin or root exist. I don't think the password matters.